### DISCUSSION TITLE
Flexible recording start and stop

### DISCUSSION THREAD

**Discussion by jmvaliente** [roleoroleo/yi-hack-Allwinner-v2#638] [Q&A] [answered]
I have seen that in the code there is a flag to modify the maximum duration of the video "-S" in "ipc_cmd.c", but I see that you can't use it by passing it by argument using "camera_settings.sh".

Is it possible to set the video duration dynamically from camera_settings?

The motivation of the change from 60 seconds to shorter duration videos is to capture a 4 or 5 seconds video where the initial movement is and to be able to treat it with AI, later.

**Comment by roleoroleo** [comment, marked as answer]
This flag can be used only when you start a recording manually through the web interface record.sh
The length of the "normal" recording is handled by Yi programs.
